templates/start.html

```diff
@@ -1,7 +1,6 @@
-{% extends "layout.html" %}
-{% block title %}{{ this.title }}{% endblock %}
-{% block body %}
-
+{%- extends "layout.html" %}
+{%- block title %}{{ this.title }}{% endblock %}
+{%- block body %}
 <!-- Banner -->
```

Comment:
der folgende <!-- Banner --> Block ist nicht korrekt eingerückt. Da fehlen circa 6 Leerzeichen (``      ``).

Aber das sind ja echt nur schönheitsfehler.

Sollte das durch dieses indentfix nicht behoben werden? Oder verwechsel ich das?

Reply:
fehlen auch noch wo anderes einrückungen. egal. besser als davor.

Reply:
ja ich weiß. Das Problem ist hier die Markdown Umwandlung von lektor. Hierbei werden code Blöcke aus markdown in ein Konstrukt aus pre und code tag umgewandelt. Dies hat zur Folgen, dass wenn ich den ganzen Block einrücken eben auch der pre tag mit eingerückt ist und dadurch nicht mehr korrekt. Dies könnte man beheben indem man einen eigenen Filter schreibt der in diesen pre tag all newlines durch <br> ersetzt. Ich bin mir nur nicht so sicher ob es sinnvoll ist da einfach mit regex replace drüberzugehen (funktioniert, aber gefällt mir nicht so richtig)

Reply:
Ich denke aber selbst wenn wir das durch eine entsprechende Anpassung machen sollten, sollte das dann eher in einen gesonderten PR, da dieser ja doch schon mehr als groß genug ist :)